app/models/ongoing_event.rb

```diff
@@ -0,0 +1,43 @@
+class OngoingEvent < ApplicationRecord
+  acts_as_paranoid
+
+  belongs_to :ngo
+  has_many   :participations, dependent: :destroy
+  has_many   :users,          through:   :participations
+
+  validates :title, length: { in: 1..100 }
+  validates :address, presence: true
+  validates :contact_person, presence: true
+
+  scope :newest_first, -> { order(:created_at) }
+  scope :published,    -> { where.not(published_at: nil) }
+
+  def pending?
```

Comment:
If the whole `state` handling is the same for `event` and `ongoing_event`, it would eventually make sense to extract it into a concern and only test it once or with a shared example. Might go out of hand pretty quick otherwise.